src/sagemaker/tensorflow/estimator.py

```diff
@@ -137,6 +137,19 @@ def __init__(
                         To find a complete list of parameters for SageMaker model parallelism,
                         see :ref:`sm-sdk-modelparallel-general`.
 
+                **To enable Multi Worker Mirrored Strategy:**
```

Comment:
would this change be needed for other frameworks?

Reply:
No, this distribution mechanism is specific to TensorFlow.